_config.yml

```diff
@@ -4,12 +4,12 @@ descriptions: A simple set of rules and requirements that dictate how version nu
 # Should be in the form of
 # <machine readable key>: <human readable label>
 languages:
-  en: english
+  en: English
   fr: français
   ru: pyccкий
   es: español
   it: italiano
-  id: indonesia
+  id: bahasa Indonesia
```

Comment:
it should "Bahasa Indonesia"

Reply:
See: https://github.com/semver/semver/issues/512
It appears that's not the case, and that it should be `bahasa Indonesia`. However, I am not a native speaker of that language, so I cannot personally confirm one way or the other.

Reply:
In English, we would say something like "I speak English", so "English" is correct in that case.
In German, we would say something like "Ich spreche Deutsch", so "Deutsch" is correct in that case.
According to Google Translate, an Indonesian would say something like "Saya berbicara bahasa indonesia", so I would think that "bahasa indonesia" is correct. However, in that comment thread I linked to, it was said that the correct word/phrase is "bahasa Indonesia" and according to you, the correct word/phrase is "Bahasa Indonesia".
Suffice it to say, I have no idea which is "truly" correct.

Reply:
One Indonesian website that uses `Bahasa Indonesia`: https://www.tiket.com/
Wikipedia on the other hand also lowercases the `bahasa`: https://en.wikipedia.org/wiki/Indonesian_language

🤷

Reply:
I am from Indonesia

Reply:
Oh, good! I'll update the commit. Thank you for helping out!